lib/rollbax/item.ex

```diff
@@ -2,33 +2,76 @@ defmodule Rollbax.Item do
   def draft(token, envt) do
     {:ok, host} = :inet.gethostname
     %{
-      "access_token" => token,
-      "data" => %{
-        "server" => %{
-          "host" => List.to_string(host)
+      access_token: token,
+      data: %{
+        server: %{
+          host: List.to_string(host)
         },
-        "environment" => envt,
-        "language" => language(),
-        "platform" => platform(),
-        "notifier" => notifier()
+        environment: envt,
+        language: language(),
+        platform: platform(),
+        notifier: notifier()
```

Comment:
Let's keep the string keys: first, this is refactoring (see above comment 😃), and second there's no need for atoms here IMO (we're building a JSON payload, so they'll get converted to strings anyways).